test/protocol/clients/BosonVoucherTest.js

```diff
@@ -746,6 +746,76 @@ describe("IBosonVoucher", function () {
       assert.equal(returnedRange.toString(), range.toString(), "Range mismatch");
     });
 
+    context("Contract owner is not owner of preminted vouchers", function () {
+      it("Ownership is transferred", async function () {
+        // Transfer ownership to rando
+        await bosonVoucher.connect(protocol).transferOwnership(rando.address);
+
+        // Old owner's balance before
+        const ownerBalanceBefore = await bosonVoucher.balanceOf(assistant.address);
```

Comment:
I'm aware of it (I spent quite some time on it this morning, since it was behaving strange when I was adding tests to #498 )

I changed it here, but TBH I'm not sure it's more legible this way. Especially if you want to reuse the same tx for both checking balance and event emission, it gets sort of wierd (as you can see in my change).

BTW, while implementing it, I noticed that test `Should withdraw all tokens when list length > 1` was missing two `await`s in front of last `expect`s so it passed even if you put any random value instead of amount. And because `changeTokenBalances` behaves a bit strange, I had to adjust that test as well (luckiliy `changeEtherBalances` works somewhat normally). Anyway, I'm not a fan of helper functions that require more work if you have a bit more advanced case than what is in their basic example.

Reply:
You are right; the old way is more legible than now IMO
but I'm happy to let as it's now on this tests